module/zfs/zvol.c

```diff
@@ -1573,10 +1613,16 @@ zvol_create_minors_impl(const char *name)
 	int error = 0;
 	fstrans_cookie_t cookie;
 	char *atp, *parent;
+	list_t minors_list;
```

Comment:
Add comments to the effect that the minors_list is used to gather all the datasets to be prefetched first and after the prefetch completes, create minors for ?